fdbserver/workloads/AtomicRestore.actor.cpp

```diff
@@ -42,7 +43,10 @@ struct AtomicRestoreWorkload : TestWorkload {
 		startAfter = getOption(options, "startAfter"_sr, 10.0);
 		restoreAfter = getOption(options, "restoreAfter"_sr, 20.0);
 		fastRestore = getOption(options, "fastRestore"_sr, false);
-		if (!fastRestore) {
+		// TODO: Currently atomic restore does not work well with encryption since we cannot
+		// seperate system key restore from the normal key restore, once we can support this (or no longer need to
+		// separate the two restores) we can remove the use of the knob
+		if (!fastRestore && !SERVER_KNOBS->ENABLE_ENCRYPTION) {
```

Comment:
shall we instead disable encryption  (set `disableEncryption = true`) in the test specs that involve atomic backups, instead of changing the logic here? Calling `addDefaultBackupRanges` is actually the correct behavior when encryption is enabled. Test spec flags are easier the manage, if we fix the underlying issue later on, and want to turn encryption back on for these tests.

Reply:
I don't want to turn off encryption for these tests just disable backing up system keys (since we can't restore system keys first)